app/assets/tailwind/application.css

```
@@ -12,7 +12,10 @@
 /* Import Custom Components */
 @import "./components/_action-banner.css";
 @import "./components/_article.css";
+@import "./components/_about.css";
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)